recipes/blas/meta.yaml

```diff
@@ -0,0 +1,35 @@
+# Some useful notes can be found at https://conda-forge.hackpad.com/BLAS-Numpy-Friends-86De62hoNdT
+
+{% set variant = os.environ.get('BLAS_VARIANT', 'openblas') %}
+
+package:
+  name: blas
+  version: 1.0
+
+build:
+  # We mustn't use the build number in the blas package - because we are defining the string,
+  # manually we would not be producing unique filenames.
+  number: 0
+  string: {{ variant }}
+  track_features:
+    # The "blas" package must always track a feature, else it will be
+    # prioritised over all others.
+    - blas_{{ variant }}
```

Comment:
No, I haven't found that to be the case. `BLAS_VARIANT=foobar conda build recipes/blas` produces the expected results. The variable *would* be needed in `script_env` if we wanted access to it in `build.sh`.